lib/galaxy/webapps/galaxy/services/dataset_collections.py

```diff
@@ -270,6 +273,8 @@ def serialize_element(dsc_element) -> DCESummary:
                     hdca_id=self.encode_id(hdca.id),
                     parent_id=self.encode_id(result["object"]["id"]),
                 )
+            else:
```

Comment:
This probably fails on collections that aren't populated, see the comment by the sentry bot.

Reply:
```suggestion
            elif result["element_type"] == DCEType.hda:
```